code/game/objects/structures/mirror.dm

```diff
@@ -130,8 +136,15 @@
 		return
 
 	var/mob/living/carbon/human/H = user
+	if(H.health <= H.maxHealth-10) // Exploit: magic mirror can fully heal you. 10 damage is a grace.
+		to_chat(H, "<span class='warning'>You must be in full condition before using the mirror!</span>")
+		return
+
+	var/available_choices = list("name", "race", "gender", "hair", "eyes")
+	if(uses_preference && !is_banned_from(user.ckey, "Appearance") && length(user.client.prefs.character_profiles_cached))
+		available_choices += "check your character list"
```

Comment:
This should be something like "body" or "character", "check your character list" is too long and breaks the consistency with the other choices